### DISCUSSION TITLE
Debug options?

### DISCUSSION THREAD

**Discussion by prosper00** [dbuezas/lgt8fx#110] [Q&A]
Does anyone have any idea about what debugging options are available for the lgt's? I just discovered gdb and spent a couple of days getting comfortable witb it (on the stm8 platcorm via openocd). Its a real game changer! However, the lgt doesnt have the same icsp as atmega, so, i don't know if there's any support for debugging at all

**Comment by SuperUserNameMan** [comment]
Have you looked into this repo already ? https://github.com/Edragon/LGT/

It all chinese, but there seems to be some specific SDK and tools.

**Comment by prosper00** [comment]
Hm, seems there's a debugger device availaboue 'out there' - the 'SWDICE mkii'. And a manual in chinese. No indication about software, or how to use the debugging capabilities though
